website/templates/bacon.html

```diff
@@ -0,0 +1,21 @@
+{% extends "base.html" %}
```

Comment:
These pages looks identical. If that's the case I recommend following the DRY principle and crate a "coming soon" template page to use it as a base for the new pages.

Reply:
Ok, let me check